apps/kit.svelte.dev/vercel.json

```diff
@@ -0,0 +1,29 @@
+{
+	"$schema": "https://openapi.vercel.sh/vercel.json",
+	"redirects": [
+		{
+			"source": "/",
+			"destination": "https://svelte.dev/",
+			"permanent": true
+		},
+		{
+			"source": "/docs/modules",
```

Comment:
we need a URL for each individual type here

Reply:
I think we're going to have to build an app which checks what the hash is and redirects it appropriately to the correct page